### DISCUSSION TITLE
Model without synchronous generators, problems viewing bus frequency

### DISCUSSION THREAD

**Discussion by amparocorvalan** [CURENT/andes#554] [General]
Hi everyone,

I am working on a model based only on renewable energy sources and I am having some problems to analyze the behavior of the system. In particular, I would like to see how my system behaves for different faults and disturbances but I am not able to see the value of the frequency at the different busbars. Can anyone help me with this or know how I could analyze the stability of such a system in ANDES?

For example, to see the frequency in the different buses I get:

---------------------------------------------------------------------------
AttributeError Traceback (most recent call last)
Cell In[132], line 1
----> 1 ss.TDS.plt.plot(sa.BusFreq.f)

AttributeError: 'NoneType' object has no attribute 'plot'.

---------------------------------------------------------------------------

If anyone can help me with this I would be very grateful.

Regards,
Amparo

**Comment by cuihantao** [comment]
Did you look through the output messages to rule out prior errors?

See if `ss.TDS.load_plotter()` helps.
